### DISCUSSION TITLE
How we can add or create game in Resource?

### DISCUSSION THREAD

**Discussion by sachinb80** [Sunbird-Ed/Community#443] [General Discussion]
I would need to understand how this part works.
How to add a game in Resource?
![image](https://user-images.githubusercontent.com/109504832/227575033-2906ee90-80ca-474e-aba1-030c835745fb.png)
![image](https://user-images.githubusercontent.com/109504832/227575078-5793b8c6-8c0e-4ef3-b897-18c6ecd9218f.png)

**Comment by Krishnaj20** [comment]
@sachinb80 - Please raise this in the coKreat discussion forums. Here is the link to the coKreat discussion forums:
https://github.com/orgs/Sunbird-coKreat/discussions

@mmanimegalai
